### ISSUE TITLE
Crash: libnamebench\config.pyc [101]: global name 'exc' is not defined in GetLatestSanityChecks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#93]
```
What steps will reproduce the problem?
1. running namebench GUI on XP pro

What is the expected output? What do you see instead?

logfile shows:

Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 102, in run
  File "libnamebench\base_ui.pyc", line 74, in PrepareNameServers
  File "libnamebench\config.pyc", line 101, in GetLatestSanityChecks
NameError: global name 'exc' is not defined

What version of the product are you using? On what operating system?
1.1 
Win XP SP3, all available Win updates installed


Please provide any additional information below.


```

Original issue reported on code.google.com by `tgoe...@gmail.com` on 8 Jan 2010 at 9:47

Attachments:
- [namebench error.png](https://storage.googleapis.com/google-code-attachments/namebench/issue-93/comment-0/namebench error.png)

**Comment by GoogleCodeExporter** [comment]
```
This bug was created in r536 (Jan 02) - in the addition of the censorship check 
data. It only occurs if there is a 
problem downloading the most recent censorship checks data.  Thanks to this bug 
report,  this bug is now fixed 
in r600. If you would still like to use namebench for now, the older 1.0.5 
version does not have this bug:

http://namebench.googlecode.com/files/namebench-1.0.5.exe

This fix will be part of the next release, though I do not have a schedule yet 
for it.

```

Original comment by `thomas.r...@gmail.com` on 8 Jan 2010 at 10:04
- Changed title: **Crash: libnamebench\config.pyc [101]: global name 'exc' is not defined in GetLatestSanityChecks**
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I have the same bug on MacOSX 10.6.2. See screenshot. namebench version 1.1
```

Original comment by `c.harle...@gmail.com` on 19 Jan 2010 at 6:53

Attachments:
- [Failnamebench.png](https://storage.googleapis.com/google-code-attachments/namebench/issue-93/comment-2/Failnamebench.png)

**Comment by GoogleCodeExporter** [comment]
```
 i am also facing the above problem . but i am using a proxy to connect to the internet
```

Original comment by `sakhil1...@gmail.com` on 23 Jan 2010 at 1:24

Attachments:
- [untitled.bmp](https://storage.googleapis.com/google-code-attachments/namebench/issue-93/comment-3/untitled.bmp)

**Comment by GoogleCodeExporter** [comment]
```
Same bug here.
OS: Ubuntu 9.10 - Kernel Linux 2.6.31-17
Python 2.6.4

Log:
> Running...
> Started thread
> Checking query interception status...
* DNS interception test failed (no response)
> Checking connection quality... [1/2]
> Checking connection quality... [2/2]
> Congestion level is 151.70X (check duration: 6068.02ms)
> Applied 3.00X timeout multiplier due to congestion: 6.0 standard, 10.5 health.
> Checking latest sanity reference
Traceback (most recent call last):
  File "/home/ricardo/Desktop/namebench-1.1/libnamebench/tk.py", line 102, in run
    self.PrepareNameServers()
  File "/home/ricardo/Desktop/namebench-1.1/libnamebench/base_ui.py", line 74, in 
PrepareNameServers
    (primary_checks, secondary_checks, censor_tests) = config.GetLatestSanityChecks()
  File "/home/ricardo/Desktop/namebench-1.1/libnamebench/config.py", line 101, in 
GetLatestSanityChecks
    except exc:
NameError: global name 'exc' is not defined
Showing popup: global name 'exc' is not defined
> global name 'exc' is not defined

```

Original comment by `frea...@gmail.com` on 25 Jan 2010 at 4:13

**Comment by GoogleCodeExporter** [comment]
```
Ditto Windows 7 32 bit
```

Original comment by `ourstan...@gmail.com` on 10 Feb 2010 at 7:12

Attachments:
- [namebench.JPG](https://storage.googleapis.com/google-code-attachments/namebench/issue-93/comment-5/namebench.JPG)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in namebench 1.2.

```

Original comment by `tstromb...@google.com` on 2 Mar 2010 at 2:17
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 154 has been merged into this issue.
```

Original comment by `helixblue` on 12 Jul 2010 at 5:15
